tests/scripts/create-dev-cluster.sh

```diff
@@ -113,7 +113,11 @@ setup_minikube_env() {
     echo "Setting up minikube env for profile '$ROOK_PROFILE_NAME' (using $minikube_driver driver)"
     $MINIKUBE delete
     $MINIKUBE start --disk-size="$MINIKUBE_DISK_SIZE" --extra-disks="$MINIKUBE_EXTRA_DISKS" --driver "$minikube_driver" -n "$MINIKUBE_NODES" $ROOK_MINIKUBE_EXTRA_ARGS
-    eval "$($MINIKUBE docker-env)"
+    if [ "$MINIKUBE_NODES" -eq 1 ]; then
+        eval "$($MINIKUBE docker-env)"
+    elif [ "$MINIKUBE_NODES" -gt 1 ]; then
+        $MINIKUBE addons enable registry
+    fi
```

Comment:
```suggestion
    elif [ "$MINIKUBE_NODES" -gt 1 ]; then
        $MINIKUBE addons enable registry
    fi
```

I'm not sure if we want to enable registry by default when the node is gt 1 as it requires registry config.
can you explain your use cases?

Reply:
When working with a multi-node setup, the only way I know to test the local changes is by pushing the image to the registry to be able to reference it inside the development cluster. Thus by default I always need the registry if there is more than one node in my cluster when doing development work, if there is any other way of doing I would appreciate if you could enlighten me. However, I agree that is definitely a nice to have and not a must in anyway

Reply:
have you tried `minikube image load my_image`?

Reply:
Thanks! This seems to work then I guess the registry is indeed not needed!